speeduino/secondaryTables.ino

```diff
@@ -88,17 +88,19 @@ void calculateSecondarySpark()
       currentStatus.advance2 = getAdvance2();
       //Spark 2 table is treated as a % value. Table 1 and 2 are multiplied together and divded by 100
       uint16_t combinedAdvance = ((uint16_t)currentStatus.advance1 * (uint16_t)currentStatus.advance2) / 100;
```

Comment:
this line is still broken, what happens if we have negative values in one or both tables. they are going to be converted to unsigned values and change to positive during this multiply

Reply:
Yep, agree completely here. If you want to make these `int16_t` values as part of this PR I'll happily pull it in.